concepts/bindings/README.md

```diff
@@ -1,12 +1,12 @@
 # Bindings
 
-Using bindings, you can trigger your app with events coming in from external systems, or invoke external systems.
-Bindings allow for on-demand, event-driven compute scenarios, and dapr bindings help developers with the following:
+Dapr provides a mechanism that can either (a) trigger your app in response to an event or (b) trigger an external system when your app does something.
```

Comment:
I would prefer to keep this description "Using bindings, you can trigger your app with events coming in from external systems, or invoke external systems:" I do not think that saying Dapr "triggers" an external system give the understanding that this is really like a call. And in v0.8. this will be able to return a response